onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -209,6 +227,313 @@ def try_replacing_upstream_output(self, upstream_output_name, output_name):
             return True
         return False
 
+    def create_dynamic_subgraph(self, input_name):
+        """
+        Create nodes for dynamic quantization of input and add them to nodes_list.
+            parameter input_name: Name of the input.
+            parameter nodes_list: new nodes are appended to this list.
+            parameter qType: type to quantize to.
+            return: scale_name, zero_point_name, scale_shape, zero_point_shape.
+        """
+        input_scale_name = input_name + "_scale"
+        input_zp_name = input_name + "_zp"
+        inputs = [input_name]
+        compute_quant_param_node = onnx.helper.make_node(
+            "ComputeQuantizationParameters",
+            inputs,
+            [input_scale_name, input_zp_name],
+            input_name + "_ComputeQuantizationParameters",
+            domain=self.compute_quantization_parameters_function.domain,
+        )
+        return input_scale_name, input_zp_name, [], [], compute_quant_param_node
+
+    def create_dynamic_subgraph_function(self, qType, symmetric):
+        if symmetric:
+            """
```

Comment:
Description (maybe a formula?) about the behavior of this subgraph would be nice.